.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: go
 go:
-  - 1.17.3
+  - 1.19.1
```

Comment:
Does K8s 1.25.1 require Go 1.19? Will we need to upgrade Porx to Go 1.19 as well? 

It seems k8s upgrades to the latest version of Go fairly frequently, and we might run into some compilation issues.

Reply:
I saw that k8s 1.25 works on 1.18 as well. When we vendor this in porx we either upgrade to 1.18 or 1.19. We are already on 1.17 in porx.

Reply:
Ok, hopefully porx compiles on 1.18, but if not, one minor bump from 1.17 -> 1.18 should be easy.